### PR TITLE
Standardize npm

### PR DESCRIPTION
**Pull request by onebytegone**

### CODE REVIEW

**Review by jthomerson — DISMISSED** [review]
Did you want us to merge this now, while #6 is still open, or wait?

**Comment by onebytegone** [comment]
@jthomerson I'm pulling this PR back. Will update once it is ready for review again.

File: _makeLambdaHandler.js [hunk 1]
```diff
@@ -1,23 +1,22 @@
 'use strict';
 
-var Q = require('q');
+var Q = require('q'),
+    log = console.log.bind(console); // eslint-disable-line no-console
```
Comment (jthomerson) [inline _makeLambdaHandler.js]: should we wait until you figure out a solution to the problem you discovered today with this same change in APIGW utils?

**Review by jthomerson — COMMENTED** [review]
@onebytegone question for you

File: _makeLambdaHandler.js (hunk 1 above)
Comment (onebytegone) [reply _makeLambdaHandler.js]: Yep, that's the reason I pulled this back in https://github.com/silvermine/lambda-utils/pull/5#issuecomment-447945357. I'm writing up an issue now. Will update this PR once that has been resolved.

Comment (onebytegone) [reply _makeLambdaHandler.js]: Issue created: https://github.com/silvermine/apigateway-utils/issues/22

**Review by jthomerson — APPROVED** [review]

**Pull request merged**